src/oemof/solph/_models.py

```diff
@@ -362,6 +362,13 @@ def _add_parent_block_variables(self):
                     elif (o, i) in self.UNIDIRECTIONAL_FLOWS:
                         for t in self.TIMESTEPS:
                             self.flow[o, i, t].setlb(0)
+
+                    if self.flows[o, i].value[self.TIMESTEPS[1]] is not None:
+                        for t in self.TIMESTEPS:
+                            self.flow[o, i, t].value = (
+                                self.flows[o, i].value[t]
+                                * self.flows[o, i].nominal_value
+                            )
```

Comment:
This statement does not seem to have any effect. Also, I did not see any hint in the documentation that assignment is possible. (However, it seems to work for the fixed entries.)

Reply:
I found examples that show the value being set that way in the documentation (https://pyomo.readthedocs.io/en/stable/working_models.html#changing-the-model-or-data-and-re-solving). Pyomo values are fixed, so thzis method seems to work. However, in the LP files, there is no starting value. Also, there is no run time difference.

As the [documentation for Pyomo variables](https://pyomo.readthedocs.io/en/stable/pyomo_modeling_components/Variables.html) says that initialisation "is particularly important for non-linear models", I feel that the starting value might be simply ignored for linear variables.